website/source/docs/secrets/aws/index.html.md

```diff
@@ -28,7 +28,7 @@ $ vault mount aws
 Successfully mounted 'aws' at 'aws'!
 ```
 
-Next, we must configure the root credentials that are used to manage IAM credentials:
+Next, we must configure the credentials that are used to manage IAM credentials:
```

Comment:
I would word it something like:

"Next, we must configure the credentials Vault uses to manage the IAM credentials generated by this secret backend:"